### PR TITLE
ENG-120 Validate Pull Request Title is prepended with Jira Issue Key

### PR DESCRIPTION
**Pull request by andyasberry**
### Tasks

- [x] Basic functionality
- [x] Verify it works in a PR somewhere https://github.com/invitation-homes/toolbelt-bff/pull/115
- [x] #5 is merged
- [x] Rebase with latest from `master`
- [x] Uprev version to `0.2.0`

### CODE REVIEW

**Review by wasimarif — APPROVED** [review]
LGTM

File: src/utils.ts [hunk 1]
```diff
@@ -168,6 +168,25 @@ export const validateCommitMessages = (
...
+<hr />
+<p>Please follow <a href="https://github.com/invitation-homes/technology-decisions/blob/main/0014-tracking-jira-issues-in-git.md">our standards</a> for PR titles.</p>
+Valid sample PR titles:
+
+  ‣ 'DDTS-112 Build new CMS'
+  ‣ 'TTEF-2 Fix React Native bug'
+  ‣ 'INTG-332 Add logging to external api'
+`;
+};
+
```
Comment (andyasberry) [inline src/utils.ts:189]: ❓ Any suggestions on if/how we could improve the comment that would get posted in the PR?

Comment (ryan-breidenbach) [reply src/utils.ts:189]: That looks good to me. Clear error message with examples.

**Review by davidaayers — APPROVED** [review]
🔥

**Pull request merged**